tests/Feature/Api/V1/CommentTest.php

```diff
@@ -139,11 +139,12 @@ public function testStore()
 
     public function testStoreFail()
     {
+        $model = Post::class;
         $this->actingAsUser('api')
             ->json('POST', "/api/v1/posts/31415/comments", $this->validParams())
             ->assertNotFound()
             ->assertJson([
-                'message' => 'No query results for model [App\\Models\\Post] 31415'
+                'message' => "No query results for model [{$model}] 31415"
```

Comment:
I think the temp variable is useless with this: `{Post::class}`

Reply:
I also tried it before. But unfortunately, in all php versions, string interpolation does not support that syntax.

Reply:
Ah ok.

I'm not a huge fan of temp variables.

I find this more elegant:
```php
'message' => sprintf('No query results for model [%s] 31415', Post::class)
``

Reply:
ping @khoinv

Reply:
I'm sorry for the late reply. I agree with you about the ugly of using temporary variables.
I fixed and pushed.